snap/README.md

```diff
@@ -1,73 +1,179 @@
+This folder contains snap packaging for the EdgeX Foundry reference implementation.
 
-The snap contains consul, mongodb, all of the EdgeX Go-based micro services from
-this repository, and three Java-based services, support-notifications and support-
-scheduler, and device-virtual. The snap also contains a single OpenJDK JRE used to
-run the Java-based services.
+The snap contains Consul, MongoDB, all of the EdgeX Go-based micro services from
+this repository, and three Java-based services, support-rulesengine and support-
+scheduler, and device-virtual, as well as Vault, Kong, Cassandra, and the two go-based 
+security micro services. The snap also contains a single OpenJDK JRE used to run 
+the Java-based services.
 
-## Installation Requirements
-The snap can be installed on any system running snapd, however for full confinement,
-the snap must be installed on an Ubuntu 16.04 LTS or later Desktop or Server, or a
-system running Ubuntu Core 16.
+The project maintains a rolling release of the snap on the `edge` channel that is rebuilt and published at least once daily through the jenkins jobs setup for the EdgeX project. You can see the jobs run [here](https://jenkins.edgexfoundry.org/view/Snap/) specifically looking at the `edgex-go-snap-{branch}-stage-snap`.
+
+The snap currently supports running on both `amd64` and `arm64` platforms. Once the project no longer depends on MongoDB as it's primary database, the snap should start working on `armhf` platforms that support snaps as well.
 
 ## Installation
-There are amd64 and arm64 versions of both releases available in the store.  You can
-see the revisions available for your machine's architecture by running the command:
 
-`$ snap info edgexfoundry`
+### Installing snapd
+The snap can be installed on any system that supports snaps. You can see how to install 
+snaps on your system [here](https://docs.snapcraft.io/t/installing-snapd/6735).
+
+However for full security confinement, the snap should be installed on an 
+Ubuntu 16.04 LTS or later Desktop or Server, or a system running Ubuntu Core 16 or later.
+
+### Installing EdgeX Foundry as a snap
+The snap is published in the snap store at https://snapcraft.io/edgexfoundry.
+You can see the current revisions available for your machine's architecture by running the command:
+
+```bash
+$ snap info edgexfoundry
+```
+
+The snap can be installed using `snap install`. To install the snap from the edge channel:
 
-The snap can be installed using this command:
+```bash
+$ sudo snap install edgexfoundry --edge
+```
 
-`$ sudo snap install edgexfoundry --channel=california/edge`
+You can specify install specific releases using the `--channel` option. For example to install the california release of the snap:
 
-**Note** - this snap has only been tested on Ubuntu 16.04 LTS Desktop/Server and Ubuntu Core 16.
+```bash
+$ sudo snap install edgexfoundry --channel=california
```

Comment:
This won't work right now, as we never promoted to stable. Let's discuss this later today.

Reply:
As discussed, this will just be changed to `delhi`.